form_urlencoded/src/lib.rs

```diff
@@ -13,11 +13,17 @@
 //! Converts between a string (such as an URL’s query string)
 //! and a sequence of (name, value) pairs.
 
+extern crate percent_encoding;
+#[macro_use]
+extern crate matches;
+
 use percent_encoding::{percent_decode, percent_encode_byte};
-use query_encoding::{self, decode_utf8_lossy, EncodingOverride};
+use query_encoding::EncodingOverride;
 use std::borrow::{Borrow, Cow};
 use std::str;
 
+pub mod query_encoding;
```

Comment:
This module was previously private and made public here. Is this intentional? If so, why?

Reply:
Yes, this is intentional. This functionality is used both in form_urlencoded and in url, so I decided to make it a public module of the new crate. I could also duplicate it, like the decode_utf8_lossy function that was already duplicated between percent_encoding and url (now form_urlencoded).